Ginger/Ginger/AutomatePageLib/AddActionMenu/WindowExplorer/Common/WindowExplorerPage.xaml.cs

```diff
@@ -1017,7 +1017,7 @@ private void GridTreeViewButton_Click(object sender, RoutedEventArgs e)
                 WindowControlsGridView.Visibility = System.Windows.Visibility.Visible;
                 if (WindowControlsGridView.DataSourceList == null || WindowControlsGridView.DataSourceList.Count == 0)
                         ShowFilterElementsPage();
-
+                RefreshControlsGrid();
```

Comment:
Please do this as last statement in else. Because if any error during Refresh controls the tooltip and icon won't be updated